CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- `Container`: remove 72px on xl viewport ([@farazatarodi](https://https://github.com/farazatarodi) in [#2615](https://github.com/teamleadercrm/ui/pull/2615))
```

Comment:
I guess you should mention padding, otherwise it's a bit weird 😅

Reply:
@lowiebenoot That is very true haha. I'll do it for the release.